ingestion/src/metadata/ingestion/source/database/hive/metadata.py

```diff
@@ -251,13 +323,20 @@ def prepare(self):
         Fetching views in hive server with query "SHOW VIEWS" was possible
         only after hive 2.2.0 version
         """
-        result = dict(self.engine.execute("SELECT VERSION()").fetchone())
-        version = result.get("_c0", "").split()
-        if version and self._parse_version(version[0]) >= self._parse_version(
-            HIVE_VERSION_WITH_VIEW_SUPPORT
-        ):
-            HiveDialect.get_table_names = get_table_names
-            HiveDialect.get_view_names = get_view_names
+        if self.engine.driver == "impala":
```

Comment:
```python
        if self.engine.driver == Dialects.Impala:
```

We'll need to import `from metadata.profiler.orm.registry import Dialects` at the top